apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -542,6 +542,7 @@ export const pageRouter = router({
       })
       return publishPageResource(
         ctx.logger,
+        ctx.gb,
```

Comment:
[question] if all we're passing in growthbook for is to determine whether singpass is enabled, shouldn't we do it at a higher level (eg: router) so that the service method can be agnostic?